packages/ui/src/header/index.tsx

```diff
@@ -48,6 +48,7 @@ export const Header: FC<HeaderProps> = ({
             <a
               className="ui-pl-2 md:ui-px-3 ui-h-8 ui-flex ui-items-center"
               href="/"
+              aria-label="Go to Storybook home"
```

Comment:
This is a link, so the "Go to..." is extraneous. And they're on the Storybook site, so just "Home" is fine.

```suggestion
              aria-label="Home"
```